### DISCUSSION TITLE
Add status bar statistics for single value, and include in undocked Query view

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1392] [Ideas]
It is difficult to read numbers in the query results because they are not formatted. For example:
![image](https://github.com/user-attachments/assets/7e7f8edd-4f41-40ee-9bb6-eced1f0819df)
Very difficult to read. You have to count zeros and places to see where commas go.

But if you highlight 2 or more values, the status bar gives much richer information:

![image](https://github.com/user-attachments/assets/1983b010-22f4-4d44-a67d-f404b486b681)

The problem is two fold:
1) You must highlight 2 or more values.  If your query returns a single value - say you just did a {COUNTROWS(table)} you cannot use this feature.
2) If your query pane is undocked and on a different screen or full screen, the status bar isn't available and you cannot see this info.

So, 3 suggestions:
1) Make the status bar work to show this info if only one value is selected.
2) Make the status bar (of this info) available in the query window itself
3) Consider adding a hover over tooltip in the query results that will show you numerical data formatted with commas.

**Comment by otykier** [comment]
All round great suggestions as usual, Ed. Thanks a lot for that!
